### DISCUSSION TITLE
random album

### DISCUSSION THREAD

**Discussion by labatts** [Taiko2k/Tauon#776] [Q&A]
Is there  a way to generate a random album in the smart playlist?

**Comment by Taiko2k** [comment]
You can shuffle albums in the playlist using the code `ra`, but theres currently no way to limit that to a selection of *n* albums.

**Comment by labatts** [comment]
Well, I am getting the desired effect by toggling the shuffle album in the main music playlist, and forwarding until it finds something that interests me, and then turning off the shuffle.  Gotta say, I am glad I finally took the plunge into this.  Tauon is a real mix between simplicity and complex features.  Thanks for coming to linux!
